### ISSUE TITLE
temporary block

### ISSUE THREAD

**Issue by jasperf** [antonioribeiro/firewall#152]
Is it also possible to temporarily block an ip address? Wordfence does for example block access to ip address for too many login requests for x hrs or days. Can this be done with this Laravel Firewall Package?

**Comment by antonioribeiro** [comment]
Not yet.

**Comment by jasperf** [comment]
@antonioribeiro understood. Thanks for putting out this package and if you can keep this issue open as a request for a future feature.

**Comment by antonioribeiro** [comment]
@jasperf, yes, tagged it as "enhancement" to do it later. Feel free to send a Pull Request for it, if you can.
